src/trees/prufer.jl

```diff
@@ -0,0 +1,96 @@
+"""
+    is_tree(g)
+
+Returns true if g is a tree: that is, a simple, connected undirected graph, with nv-1 edges (nv = number of vertices). Trees are the minimal connected graphs; equivalently they have no cycles.  
+
+This function does not apply to directed graphs. Directed trees are sometimes called [polytrees](https://en.wikipedia.org/wiki/Polytree)). 
+
+"""
+
+function is_tree end
```

Comment:
```suggestion
"""
function is_tree end
```
as far as I know, we can't have an empty line here - correct me if I am not right.

Reply:
I really don't know!

Reply:
Apparently it says here, that blanc lines are not allowed: https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation

And yes, this might influence the documentation that is created by Documenter.jl (and also the documentation in the REPL when you type `?is_tree`).